plugins/dfu/dfu.quirk

```diff
@@ -334,38 +334,48 @@ DfuForceTimeout = 5000
 [DeviceInstanceId=USB\VID_095D&PID_9217]
 Plugin = dfu
 Flags = manifest-poll
+DfuRemovalDelay = 35000
```

Comment:
I'm confused; can't we use the existing `RemoveDelay` quirk key for this?

Reply:
I didn't notice we already have a quirk key. I will try it. Thanks.

Reply:
Yeah, the RemoveDelay quirk key works quite well. I will remove the unnecessary changes.